### PR TITLE
Opticon Copy Edits

### PR DESCRIPTION
**Pull request by stewsmith**
### This PR does the following:
1. Moves /about to /event-info
2. Adds a link to the 2014 page on /opticon
3. Renames Agenda to Sessions. The page remains as /agenda though per Heather's request.

### CODE REVIEW

File: website/opticon/event-info/index.hbs [hunk 1]
```diff
@@ -23,13 +23,15 @@ external_script:
     <h3 class="description-title adelle-sans">{{top_paragraph.title}}</h3>
     <p class="description-text adelle-sans">{{top_paragraph.blurb}}</p>
     <a class="opticon-button adelle-sans" href="http://pages.optimizely.com/rs/optimizely/images/Opticon_2015_Justify_Your_Trip.pdf">{{top_paragraph.cta}}</a>
+    {{/with}}
     <p class="adelle-sans description-text">
-      {{top_paragraph.cta_2014}}
-      <a href="{{linkPath}}/opticon/opticon-2014">{{top_paragraph.cta_2014_link_text}} &raquo;</a>
+      {{page_data.top_paragraph.cta_2014}}
+      <a href="{{linkPath}}/opticon/opticon-2014">{{page_data.top_paragraph.cta_2014_link_text}} &raquo;</a>
```
Comment (dtothefp) [inline website/opticon/event-info/index.hbs]: @stewsmith looks like you can carry the `{{#with}}` block all the way down to just above the partial

**Comment by dtothefp** [comment]
@stewsmith should the SEO title for /agenda be renamed to sessions?

File: website/opticon/event-info/index.hbs (hunk 1 above)
Comment (stewsmith) [inline website/opticon/event-info/index.hbs]: I had to end the `{{#with}}` before `{{linkPath}}` otherwise `{{linkPath}}` is undefined

**Comment by stewsmith** [comment]
@dtothefp Good call on the SEO title

**Comment by dtothefp** [comment]
@stewsmith LGTM

**Pull request merged**